addon/components/copy-button.js

```diff
@@ -13,10 +13,16 @@ export default Ember.Component.extend({
     'clipboardTarget:data-clipboard-target',
     'clipboardAction:data-clipboard-action',
     'buttonType:type',
-    'disabled'
+    'disabled',
+    'aria-label'
   ],
 
   /**
+   * @property {String} aria-label - aria-label attribute for button element
+   */
+  'aria-label': '',
```

Comment:
Does it make sense to have a meaningful default value? If not, can we remove the property and just leave the    attribute binding?